File: app/src/main/java/com/akexorcist/lovelyrecyclerview/adapter/holder/SummaryViewHolder.java
package com.akexorcist.lovelyrecyclerview.adapter.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.akexorcist.lovelyrecyclerview.R;


/**
 * Created by Akexorcist on 10/28/2016 AD.
 */

public class SummaryViewHolder extends RecyclerView.ViewHolder {
    public TextView tvSummaryName;
    public TextView tvSummaryPrice;

    public SummaryViewHolder(View itemView) {
        super(itemView);
        tvSummaryName = (TextView) itemView.findViewById(R.id.tv_summary_name);
        tvSummaryPrice = (TextView) itemView.findViewById(R.id.tv_summary_price);
    }
}

File: app/src/main/java/com/akexorcist/lovelyrecyclerview/adapter/holder/SummaryViewHolder.kt
package com.akexorcist.lovelyrecyclerview.adapter.holder

import androidx.recyclerview.widget.RecyclerView
import com.akexorcist.lovelyrecyclerview.adapter.model.OrderDetailItem
import com.akexorcist.lovelyrecyclerview.databinding.ViewSummaryBinding

class SummaryViewHolder(private val binding: ViewSummaryBinding) : RecyclerView.ViewHolder(binding.root) {
    fun bind(item: OrderDetailItem.Summary) {
        binding.tvSummaryName.text = item.name
        binding.tvSummaryPrice.text = item.price
    }
}
